docs/source/user/runtime_view.rst

```diff
@@ -172,6 +180,43 @@ level has its own corresponding method. The second example shows how to use
 the more general ``log`` method. This particular overload of the ``log`` method
 allows you to specify (at runtime if you like) the severity of the message.
 
+********************
+Python Example
+********************
+
+.. tabs::
+
+   .. tab:: Python
+
+      .. code-block:: python
+
+         import parallelzone as pz
+
+         log = pz.runtime.RuntimeView().logger()
+         severity = pz.Logger.severity
+         severities = [severity.trace, severity.debug, severity.info, severity.warn, severity.error, severity.critical] 
+
+         log.set_severity(severity.critical)
+
+         for level in severities:
+            log.log(level, "Hello")
+
+         # OUTPUT:
+         # [2025-05-06 11:29:54.722] [Rank 0] [critical] Hello
+
+         log.set_severity(severity.trace)
+
+         for level in severities:
+            log.log(level, "Hello")
+
+         # OUTPUT:
+         # [2025-05-06 11:33:05.203] [Rank 0] [trace] Hello
+         # [2025-05-06 11:33:05.203] [Rank 0] [debug] Hello
+         # [2025-05-06 11:33:05.203] [Rank 0] [info] Hello
+         # [2025-05-06 11:33:05.203] [Rank 0] [warning] Hello
+         # [2025-05-06 11:33:05.203] [Rank 0] [error] Hello
+         # [2025-05-06 11:33:05.203] [Rank 0] [critical] Hello
+
```

Comment:
Please use literalinclude and ensure your code is tested. Please also explain what your code is showing (in your case, it illustrates the difference in output when using different severities).